hw/ip/sysrst_ctrl/dv/sysrst_ctrl_sim_cfg.hjson

```diff
@@ -106,6 +106,11 @@
       uvm_test_seq: sysrst_ctrl_edge_detect_vseq
       run_opts: ["+test_timeout_ns=10_000_000_000"]
     }
+    {
+      name: sysrst_ctrl_combo_detect_with_pre_cond
+      uvm_test_seq: sysrst_ctrl_combo_detect_with_pre_cond_vseq
+      reseed: 80
```

Comment:
After an initial run, I estimate that we need at least 500 reseeds to hit all the cover points in: https://github.com/lowRISC/opentitan/pull/17337
Tomorrow I will do a longer run with 500 iterations to confirm that this is enough.

Reply:
After the recent changes to the covergroup, I think we should increase these iterations to 100. We can always fine tune this later.